### ISSUE TITLE
How might you call console.log?

### ISSUE THREAD

**Issue by ctaggart** [neon-bindings/neon#226]
`console` is a [global object in Node.js](https://nodejs.org/api/globals.html). How would I get a reference to it, so that I could call the `.log` function`? 

Background: I want to play with a the [TypeScript library from Neon](https://github.com/ctaggart/TsAst/issues/10), but need help getting started with Neon.

**Comment by ctaggart** [comment]
@dherman, pretty please.

**Comment by ctaggart** [comment]
I closed this thinking that I would be able to use the chakracore-rs library, but that [didn't work out](https://github.com/darfink/chakracore-rs/issues/21#issuecomment-327452792). I'm reopening this, hoping that I can get some help learning to use it.

**Comment by dherman** [comment]
OK, the API that's missing is the ability to get to the current global object -- filed as #249. From there you can pull off the `console` object and extract the `log` method. (We should also add conveniences to eliminate some of the tedium of all those steps.)

**Comment by dherman** [comment]
Fixed in #253:

```rust
pub fn call_console_log(call: Call) -> JsResult<JsObject> {
    let scope = call.scope;
    
    let global = scope.global();
    let console = global.get(scope, "console")?.check::<JsObject>()?;
    let log = console.get(scope, "log")?.check::<JsFunction>()?;
    let args: Vec<Handle<JsString>> = vec![JsString::new(scope, "hello world").unwrap()];
    log.call(scope, console, args.into_iter())?;
    // ...
}
```
